src/test/java/com/researchspace/protocolsio/PIOComponentParsingTest.java

```diff
@@ -19,7 +20,7 @@
 import com.fasterxml.jackson.databind.JsonMappingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
-public class TestP_io {
+public class PIOComponentParsingTest {
```

Comment:
with Jupiter tests, test methods and class doesn't need to be public, just reduces a bit of clutter